t3/utils/flux.py

```diff
@@ -322,17 +322,21 @@ def run_jsr(gas: ct.Solution,
         dict: The T, P, X, and ROP profiles (values) at a specific time.
     """
     V = V or 100
+    num_steps = 500
     profiles = dict()
     stoichiometry = get_rxn_stoichiometry(gas)
     for tau in times:
-        network, reactor = set_jsr(gas=gas, time=tau, composition=composition, T=T, P=P, V=V, a_tol=a_tol, r_tol=r_tol)
         t = 0
-        while t < tau:
+        dt = tau/num_steps
```

Comment:
minor: please add a space before and after operators, e.g. `tau / num_steps`
Also below for `tau*tau_tolerance` and other cases